packages/itwinui-css/src/color-picker/color-picker.scss

```diff
@@ -73,12 +72,12 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(var(--iui-color-foreground-body
   display: flex;
   flex-wrap: wrap;
   align-content: flex-start;
-  min-height: $iui-l;
-  min-width: $iui-l;
-  gap: $iui-s;
+  min-height: var(--iui-size-l);
+  min-width: var(--iui-size-l);
+  gap: var(--iui-size-xs);
```

Comment:
```suggestion
  gap: var(--iui-size-xs);
  align-items: center;
```
The borderless button & swatches are not aligned in this branch:
![Screen Shot 2022-08-12 at 10 12 00 AM](https://user-images.githubusercontent.com/849817/184371479-655c015e-47e4-4f8c-8339-96e91522acea.png)

Reply:
Fixed